aws-ecsfargate-terraform/main.tf

```diff
@@ -296,3 +296,8 @@ resource "aws_route53_record" "op_scim_bridge" {
     evaluate_target_health = true
   }
 }
+
+moved {
+  from = aws_secretsmanager_secret_version.scimsession_1
+  to   = aws_secretsmanager_secret_version.scimsession
+}
```

Comment:
Can you tell me more about this structure? What is this communicating to the terraform system? That both filenames are valid, and to rename one to the other?

Reply:
It's a method for cleanly refactoring Terraform code that will help us improve our deployment example with less disruption for existing customers:
https://www.terraform.io/language/modules/develop/refactoring#moved-block-syntax

Reply:
Fantastic. Thank you.